portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -46,26 +50,39 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     PreparedQuery results = datastore.prepare(query);
 
     ArrayList<String> comments = new ArrayList<String>();
```

Comment:
nit: ArrayList<String> comments -> List<String> comments
also for emails.

Reply:
Ack. Changing it now